src/ui/settings/abstract-settings-ui-handler.ts

```diff
@@ -411,4 +462,10 @@ export default class AbstractSettingsUiHandler extends UiHandler {
     }
     this.cursorObj = null;
   }
+
+  showText(text: string, delay?: number|null, callback?: Function, callbackDelay?: number, prompt?: boolean, promptDelay?: number): void {
+    this.settingsMessageBoxContainer.setVisible(!!text);
```

Comment:
nit: code style preference
```suggestion
    this.settingsMessageBoxContainer.setVisible(text.length > 0);
```